components/PDFViewer.jsx

```diff
@@ -15,6 +15,8 @@ const PDFViewer = ({ pdf }) => {
   };
 
   useEffect(() => {
+    pdf =
+      "https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf";
     if (pdf) {
       if (pdf.endsWith(".pdf")) {
```

Comment:
you can combine the if statements

Reply:
This doesn't work for google drive links

Reply:
fixed by searching for "drive.google.com" in string

Reply:
![image](https://github.com/acm-ucr/membership-portal/assets/53993828/f8e8c26e-bd1c-43d7-a2eb-8a3f6f3bbf54)